app/javascript/pages/account/PasswordFieldset.tsx

```diff
@@ -1,15 +1,28 @@
 import { Field, passwordRegex, t } from "@bloom-housing/ui-components"
 import React from "react"
 import { UseFormMethods } from "react-hook-form"
-import "./password-fieldset.scss"
+import "./account-settings"
+import Fieldset from "./Fieldset"
 
 const NewPasswordInstructions = () => {
   return (
-    <ul className="password-instructions">
-      <li>{t("createAccount.passwordInstructions.numCharacters")}</li>
-      <li>{t("createAccount.passwordInstructions.numLetters")}</li>
-      <li>{t("createAccount.passwordInstructions.numNumbers")}</li>
-    </ul>
+    <>
+      <span>{t("createAccount.passwordInstructions.mustInclude")}</span>
+      <ul className="password-instructions">
```

Comment:
Is it possible to use tailwind's [list-disc](https://tailwindcss.com/docs/list-style-type#basic-usage) class to add the bullets to the list?